pkg/kmutex/kmutex.go

```diff
@@ -68,15 +81,19 @@ func (m *KMutex) Trylock(k string) bool {
 		return true
 	}
 
+	// the k has already been locked.
 	select {
 	case <-v.c:
+		// the locker has released the lock.
 		return true
```

Comment:
In this line, we think there is a bug there.
If this goroutine gets the lock and returns true, there is a very tiny possibility that the GC function would delete the corresponding locked key.